herald/migrations/0006_auto_20170825_1813.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.8 on 2017-08-25 23:13
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('herald', '0005_merge_20170407_1316'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='sentnotification',
+            name='user',
+            field=models.ForeignKey(default=None, null=True, on_delete=django.db.models.deletion.SET_NULL, to=settings.AUTH_USER_MODEL),
+        ),
+    ]
```

Comment:
These migrations should be squashed. No need to have 5 new migrations for this release